parkings/tests/api/internal/test_parking.py

```diff
@@ -0,0 +1,62 @@
+import json
+
+from django.core.urlresolvers import reverse
+
+from ..utils import check_disallowed_methods, check_list_endpoint_base_fields, get
+
+list_url = reverse('internal:v1:parking-list')
+
+
+def get_detail_url(obj):
+    return reverse('internal:v1:parking-detail', kwargs={'pk': obj.pk})
+
+
+def check_parking_data(parking_data, parking_obj):
+    """
+    Compare parking data dict returned from the API to the actual Parking object.
+    """
+
+    # check keys
+    all_fields = {'id', 'created_at', 'modified_at', 'address', 'device_identifier', 'location', 'operator',
+                  'registration_number', 'resident_code', 'special_code', 'time_start', 'time_end', 'zone'}
+    assert set(parking_data.keys()) == all_fields
+
+    # string valued fields should match 1:1
+    for field in {'device_identifier', 'registration_number', 'resident_code', 'special_code', 'zone'}:
+        assert parking_data[field] == getattr(parking_obj, field)
+
+    assert parking_data['id'] == str(parking_obj.id)
+    assert parking_data['created_at'] == parking_obj.created_at.strftime('%Y-%m-%dT%H:%M:%S.%fZ')
+    assert parking_data['modified_at'] == parking_obj.modified_at.strftime('%Y-%m-%dT%H:%M:%S.%fZ')
+    assert parking_data['time_start'] == parking_obj.time_start.strftime('%Y-%m-%dT%H:%M:%SZ')
+    assert parking_data['time_end'] == parking_obj.time_end.strftime('%Y-%m-%dT%H:%M:%SZ')
+    assert parking_data['operator'] == str(parking_obj.operator_id)
+    assert parking_data['location'] == json.loads(parking_obj.location.geojson)
+    assert parking_data['address'] == str(parking_obj.address_id)
+
+
+def test_list_endpoint_base_fields(staff_api_client):
+    parking_data = get(staff_api_client, list_url)
+    check_list_endpoint_base_fields(parking_data)
+
+
+def test_disallowed_methods(staff_api_client, parking):
+    disallowed_methods = ('post', 'put', 'patch', 'delete')
+    urls = (list_url, get_detail_url(parking))
+    check_disallowed_methods(staff_api_client, urls, disallowed_methods)
+
+
+def test_get_list_check_data(staff_api_client, parking):
+    data = get(staff_api_client, list_url)
+    assert len(data['results']) == 1
+    check_parking_data(data['results'][0], parking)
+
+
+def test_get_detail_check_data(staff_api_client, parking):
+    parking_data = get(staff_api_client, get_detail_url(parking))
+    check_parking_data(parking_data, parking)
+
+
+def test_staff_required_to_get_list_or_detail(operator_api_client, parking):
+    operator_api_client.get(list_url, status_code=403)
```

Comment:
As discussed IRL, this probably wants to use `get` not  `operator_api_client.get`.